airflow/dags/create_external_tables/README.md

```diff
@@ -3,3 +3,46 @@
 Type: [Now / Scheduled](https://docs.calitp.org/data-infra/airflow/dags-maintenance.html)
 
 This DAG orchestrates the creation of [external tables](https://cloud.google.com/bigquery/docs/external-data-sources), which serve as the interface between our raw / parsed data (stored in Google Cloud Storage) and our data warehouse (BigQuery). Most of our external tables are [hive-partitioned](https://cloud.google.com/bigquery/docs/hive-partitioned-loads-gcs).
+
+Here is an annotated example external table YAML file showing what the fields mean:
+
+```yaml
+# throughout this example, <> brackets denote sample content to be filled in based on your use case and should be removed 
+operator: operators.ExternalTable   # the name of the operator; this does not change
+bucket: gs://<your bucket name>    # fill in the name of your source data bucket here
+prefix_bucket: true     # Boolean for whether or not the bucket name should have `test-` appended when you're running from local Airflow (use this if there's a `test-` bucket used for testing)
+post_hook: |    # this is optional; can provide an example query to check that external table was created successfully. this query will run every time the external table DAG runs
+  SELECT *
+  FROM `{{ get_project_id() }}`.<your dataset as defined below under destination_project_dataset_table>.<your table name as defined below under destination_project_dataset_table>
+  LIMIT 1;
+source_objects: # this tells the external table which path to look in for the objects that will be queryable through this external table  
+  - "<the top level folder name within your bucket that should be used for this external table like my_data>/*.jsonl.gz"     
+destination_project_dataset_table: "<desired dataset name like external_my_data_source>.<desired table name, may be like topic_name__specific_data_name>"   # this defines the external table name (dataset and table name) through which the data will be accessible in BigQuery
+source_format: NEWLINE_DELIMITED_JSON   # file format of raw data; generally should not change
```

Comment:
Where can somebody view a canonical list of allowable values for this field? I know it's rarely anything else, but it seems worth pointing to where this is defined or limited, if such a resource exists.